KsApi/models/graphql/UserCreditCardsTests.swift

```diff
@@ -12,11 +12,11 @@ final class GraphUserCreditCardTests: XCTestCase {
     let data = jsonString.data(using: .utf8)
 
     do {
-      let cards = try JSONDecoder().decode(GraphUserCreditCard.self, from: data!)
+      let cards = try JSONDecoder().decode(UserCreditCards.self, from: data!)
 
       XCTAssertEqual(cards.storedCards.count, 0)
     } catch {
-      XCTFail("Failed to decode GraphUserCreditCard")
+      XCTFail("Failed to decode UserCreditCards")
```

Comment:
Also shouldn't be flagging a test file here. Can you look into this and see why codecov is flagging errantly?

Reply:
Ok, previously the `.codecov.yml` didn't ignore any files, my hunch is that adding `*Tests.swift` to the top level header `.ignore` will remove reporting coverage for the test files themselves. I will push that commit up shortly.

Reply:
Let's see if that clears these warnings.

Reply:
No warnings seem to appear on test files anymore and the coverage didn't drop, so that's a good sign.

Reply:
Oh wait this is still here.